eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -45,6 +45,12 @@ parameters:
       Canary:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
         Location: 'centraluseuap'
+        MatrixFilters:
+          - OSVmImage=.*Ubuntu.*
+          - NodeTestVersion=10.x
+          - DependencyVersion=^$
+        MatrixReplace:
+          - NodeTestVersion=10.x/14.x
```

Comment:
Do we not already have 14.x included? If not why are we filtering on 10.x and then replacing it? Why not add 14.x or use a version we are testing in the canary runs?

Reply:
@HarshaNalluru asked if we could cover a different scenario for the canary tests than is covered in the base for public (which does windows+14.x), hence why I'm swapping the version here. I think there's a counter-argument for running something in canary that we already run in public, so if we see regressions we can know they are related to canary and not the unique os+version combination.

Reply:
Hmm.. makes more sense. I like the counter-argument.